### PR TITLE
relaton/support#7 move to external workflow

### PR DESCRIPTION
**Pull request by CAMOBAP**
- relaton/support#7

### CODE REVIEW

**Comment by CAMOBAP** [comment]
@andrew2net could you please suggest relevant ruby code here to replace `system` call

File: crawler.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'fileutils'
+
+t1 = Time.now
+puts "Started at: #{t1}"
+
+FileUtils.rm_rf("data")
+system("relaton fetch-data ietf-rfc-entries")
```
Comment (andrew2net) [inline crawler.rb]: @CAMOBAP let's use `RelatonIetf::DataFetcher.fetch("ietf-rfc-entries")` instead call `system`

File: crawler.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+t2 = Time.now
+puts "Stopped at: #{t2}"
+puts "Done in: #{(t2 - t1).round} sec."
```
Comment (andrew2net) [inline crawler.rb]: please remove the time start/stop outputs

File: Gemfile [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+source "https://rubygems.org"
+
+gem 'relaton-cli'
```
Comment (andrew2net) [inline Gemfile]: replace relaton-cli with relaton-ietf

File: crawler.rb [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'fileutils'
+
```
Comment (andrew2net) [inline crawler.rb]: add `require 'relaton-ietf'

**Review by andrew2net — CHANGES_REQUESTED** [review]

**Review by andrew2net — APPROVED** [review]

**Review by andrew2net — APPROVED** [review]

**Pull request merged**